Automattic/wp-calypso
# Magic Login: Design tweaks

This PR improves the design of the magic login experience.

* Put all text inside the card for better readability.
* Improve accessibility with a label for the email input.
* Move error notice below the title for consistency with other flows.
* Update the email successful screen with a new style illustration and more consistency between screens.
* Various copy updates and small visual tweaks.

Before | After
------------ | -------------
![image](https://user-images.githubusercontent.com/448298/28891234-d3ff9ac6-7797-11e7-9a70-e51eb23ee55c.png) | ![image](https://user-images.githubusercontent.com/448298/28891188-a4eb1f8a-7797-11e7-89b8-0575d4952b63.png)
![image](https://user-images.githubusercontent.com/448298/28891256-e54afec4-7797-11e7-9aea-c467cc45f18c.png) | ![image](https://user-images.githubusercontent.com/448298/28891211-b9ce3982-7797-11e7-8e18-2df5045c36da.png)
![image](https://user-images.githubusercontent.com/448298/28891326-15ec650e-7798-11e7-9b11-3464d11c5a1e.png) | ![image](https://user-images.githubusercontent.com/448298/28891168-91c2da2e-7797-11e7-97c7-eec4178bebb2.png)

#### Testing

1. Run `git checkout update/magic-login-design-tweaks` and start your server, or open the [live branch](https://calypso.live/?branch=update/magic-login-design-tweaks).
2. Visit the the magic link start at [`/log-in/link`](http://calypso.localhost:3000/log-in/link).
3. Assert the screens look like the above screenshots.
4. Try an email address that isn't associated with an account.
5. Try an email address that does have an account.
5. Make sure everything works as expected.
